include/time_metrics.h

```diff
@@ -75,7 +75,11 @@ class TimeMetricsProcessor {
     for(const auto& name : allTimings) {
       if(unavailableTimings.count(name) == 0) {
         std::vector<double> resultsSeconds;
-        std::transform(timingResults.at(name).begin(), timingResults.at(name).end(), std::back_inserter(resultsSeconds),
+        auto times_begin = timingResults.at(name).begin();
+        if (args.warmup_run) {
+          ++times_begin;
+        }
+        std::transform(times_begin, timingResults.at(name).end(), std::back_inserter(resultsSeconds),
```

Comment:
Does this lead to the first run to be omitted from the runtime sample or would it still be included in the samples list?

Reply:
Totally omitted :+1: